pkg/steps/input_image_tag.go

```diff
@@ -85,9 +89,13 @@ func (s *inputImageTagStep) Run(ctx context.Context, dry bool) error {
 	}
 
 	if len(s.config.BaseImage.Cluster) > 0 && s.srcClient != s.dstClient {
-		from, err := istObjectReference(s.srcClient, s.config.BaseImage)
-		if err != nil {
-			return fmt.Errorf("failed to reference source image stream tag: %v", err)
+		from := coreapi.ObjectReference{Kind: "DockerImage", Name: fmt.Sprintf("registry.svc.ci.openshift.org/%s/%s@sha256:SHA",
```

Comment:
This is ugly but @petr-muller  @alvaroaleman I'm not sure we lose anything, really, from this.

Reply:
That doesn't really bother me